app.rb

```diff
@@ -1,7 +1,28 @@
 require 'sinatra/base'
 class RockPaperScissors < Sinatra::Base
-  get '/test' do
-    'test page'
+  get '/' do
+    erb :index
   end
+
+  post '/name' do
+    #pass the name the user has submitted to the instance variable
+    @player_name = params[:player_name]
+    erb :choose_weapon
+  end
+
+  get '/choose_weapon' do
+    #pass the weapon chosen to the instance variable
```

Comment:
I know we chatted on Monday about next steps here - have you been able to make any further progress?